geoportailv3/static/js/draw/featurepopupdirective.js

```diff
@@ -111,8 +161,35 @@ app.FeaturePopupController.prototype.initForm_ = function() {
 app.FeaturePopupController.prototype.validateModifications = function() {
   this.feature.set('name', this.tempName);
   this.feature.set('description', this.tempDesc);
+  this.drawnFeatures_.saveFeature(this.feature);
   this.editingAttributes = false;
 };
 
 
+/**
+ * Puts the temporary edited name and description back to the feature.
+ * @export
+ */
+app.FeaturePopupController.prototype.deleteFeature = function() {
+  this.appFeaturePopup_.hide();
+  this.selectedFeatures_.remove(this.feature);
+  this.drawnFeatures_.remove(this.feature);
+  this.deletingFeature = false;
+};
+
+
+/**
+ * returns a trusted html content
+ * @param {string} content content to be trusted
+ * @return {*} the trusted content.
+ * @export
+ */
+app.FeaturePopupController.prototype.trustAsHtml = function(content) {
+  if (!goog.isDefAndNotNull(content)) {
+    content = '';
+  }
+  return this.sce_.trustAsHtml('' + content);
```

Comment:
What purpose does `'' +` serve here? `content` is a string already, so why prepending `''`?